tests/test_addplot.py

```diff
@@ -0,0 +1,153 @@
+import os
+import os.path
+import glob
+import mplfinance        as mpf
+import matplotlib.pyplot as plt
+from   matplotlib.testing.compare import compare_images
+
+print('mpf.__version__ =',mpf.__version__)                 # for the record
+print("plt.rcParams['backend'] =",plt.rcParams['backend']) # for the record
+
+base='addplot'
+tdir = os.path.join('tests','test_images')
+refd = os.path.join('tests','reference_images')
+
+globpattern = os.path.join(tdir,base+'*.png')
+oldtestfiles = glob.glob(globpattern)
+
+for fn in oldtestfiles:
+    try:
+        os.remove(fn)
+    except:
+        print('Error removing file "'+fn+'"')
+
+IMGCOMP_TOLERANCE = 7.0
```

Comment:
```suggestion
IMGCOMP_TOLERANCE = 11.0
```
Needs to be at least 11.0 for all of the image comparisons to work.